docs/commands.md

```diff
@@ -109,6 +109,11 @@ cpl copy-image-from-upstream -a $APP_NAME --upstream-token $UPSTREAM_TOKEN --ima
 - Deletes the whole app (GVC with all workloads, all volumesets and all images) or a specific workload
 - Also unbinds the app from the secrets policy, as long as both the identity and the policy exist (and are bound)
 - Will ask for explicit user confirmation
+- Optionally runs a pre-deletion hook before the app is deleted,
```

Comment:
Changes in this file are automatically generated based on the command files, so see those instead.